tests/Panel/PageTest.php

```diff
@@ -276,6 +276,55 @@ public function testImage()
         $this->assertTrue(Str::endsWith($image['url'], '/test.jpg'));
     }
 
+    /**
+     * @covers ::imageSource
+     * @covers \Kirby\Panel\Model::image
+     * @covers \Kirby\Panel\Model::imageSource
+     */
+    public function testImageCover()
+    {
+        $page = new ModelPage([
+            'slug' => 'test',
+            'files' => [
+                ['filename' => 'test.jpg']
+            ]
+        ]);
+
+        $hash = $page->image()->mediaHash();
+
+        // cover disabled as default
+        $this->assertSame([
+            'ratio' => '3/2',
+            'back' => 'pattern',
+            'cover' => false,
+            'url' => '/media/pages/test/' . $hash . '/test.jpg',
+            'cards' => [
+                'url' => 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw',
+                'srcset' => '/media/pages/test/' . $hash . '/test.jpg 352w, /media/pages/test/' . $hash . '/test.jpg 864w, /media/pages/test/' . $hash . '/test.jpg 1408w'
+            ],
+            'list' => [
+                'url' => 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw',
+                'srcset' => '/media/pages/test/' . $hash . '/test.jpg 38w, /media/pages/test/' . $hash . '/test.jpg 76w'
+            ]
+        ], (new Page($page))->image());
+
+        // cover enabled
+        $this->assertSame([
+            'ratio' => '3/2',
+            'back' => 'pattern',
+            'cover' => true,
+            'url' => '/media/pages/test/' . $hash . '/test.jpg',
+            'cards' => [
+                'url' => 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw',
+                'srcset' => '/media/pages/test/' . $hash . '/test.jpg 352w, /media/pages/test/' . $hash . '/test.jpg 864w, /media/pages/test/' . $hash . '/test.jpg 1408w'
```

Comment:
@lukasbestle Doesn't it seem strange to you that there is no width and height information at the end of the file? I can see it at the end of the image files when I enter the panel. 

For example srcset: `/media/pages/photography/sky/1969c671f4-1617089282/dark-forest-352x.jpg 352w`?